### DISCUSSION TITLE
mrv2 v0.8.0

### DISCUSSION THREAD

**Discussion by ggarra13** [ggarra13/mrv2#135] [Announcements]
v0.8.0
=====

- Fixed mrv2's file requester not selecting files with [] in them.
- Fixed mrv2's file requester not changing directories if you had typed the
  full name of the directory in the filename field and pressed Enter,
- Fixed a crash when loading a session with no files.
- Added session name to the window's title bar.
- Fixed a typo in Python's binding to session (oepenSession instead of
  openSession).
- Made Save Session not save temporary EDLs in the session file.
- Added a '\_\_divider\_\_' tuple entry to Plug-in menus to add a divider line
  between menu entries.
- Made Python's output and errors automatically be sent to the Python editor,
  instead of waiting until the commands finish, like in v0.7.9 and previous
  ones.
- Added a cmd.getVersion() to get the version of mrv2 from Python.
- Made playback play with audio when changing frame rate (slower or faster).
- Made audio play when stepping through frames.  It is currently a hack and
  not a proper fix yet.  Also, the stepping buttons are not updated properly.
- Fixed a locale change when using the FPS pull-down and there were thumbnails
  present.
- Fixed macOS menu bar font size when switching from macOS menus back to
  normal ones.
- Made saving of .otio files also work from File/Save/Movie or Sequence if the
  extension given is .otio.
- Added user metadata to save in the session file as "metadata".  This can be
  set with the Python commands setSessionMetadata and retrieved with
  sessionMetadata.
- Added a warning check when saving a session with temporary EDLs in it.
- Added timeline.speed(), timeline.defaultSpeed() and timeline.setSpeed() to
  retrieve and manipulate the FPS.
- Added image.ocioIcs() and image.setOcioIcs() and image.ocioIcsList() to
  Python to set the input color space of the image.
- Added image.ocioView(), image.setOcioView() and image.ocioViewList() to
  Python to set the Display/View color space and to retrieve a list of all
  Display/Views.
- Fixed reading of OCIO file name in network connections.
- Color channels (layers) are now kept with the file so that switching between
  media will not revert to the rgba channel if there isn't an equivalent one.
- USD Panel is now interactive.  You can change the parameters and it will
  show the change.  The only parameter not recommended to change (except for
  very simple scenes) is the complexity.
- USD Panel visibility is now saved in the Preferences.
- Refreshing of cache is now done in seconds, without re-loading and
  switching an image as before.
- Fixed a crash when creating an empty timeline or a timeline from a clip in
  the Playlist Panel.
- Fixed missing frames (Repeat Last and Repeat Scratched) when the user was
  reading a different layer and he was playing backwards or stepping through
  the frames.
- Added drawing background as transparent, solid or checkers.
- Made dragging a clip from the Files Panel not loose the selection.
- Fixed a network error (harmless) about edit mode.
- Fixed Creation of EDL Playlist with image sequences.
- Fixed annotations copying from source clip to EDL Playlist when adding the
  clip to the playlist.
- Adding a clip to an EDL playlist will now positiong the current time at the
  frame of the new clip, instead of resetting it to 0.
- Session files also save and load the timeline viewport options (ie. Edit mode,
  size of thumbnails, transitions and markers).
- Thumbnail above the timeline no longer appears when there's no clip loaded
  after it was shown once.
- Fixed pixel zooming and panning on timeline viewport on network connections
  when the pixels per unit was not 1 (like macOS Retina).
- Added a Reset Hotkeys to default in the Hotkeys Window.  This will reset
  *all* your hotkeys to the default values of mrv2 (no need to mess with
  mrv2.keys.prefs).
- Added a Reload Hotkeys to the Hotkeys Window.  This will reload the last saved
  hotkeys (ie. $HOME/.filmaura/mrv2.keys.prefs).
- Fixed Annotations Clear Frame and Annotations Clear All Frames hotkeys being
  the same.
- Hotkeys are now compared properly when they are uppercase and shift was not
  used while another hotkey had shift and a the same lowercase letter.
- Some UI fixes and improvements:
    * The Zoom factor in the Pixel Toolbar keeps its value when selecting
      it from the pulldown.
    * All buttons and displays have the same size on both the timeline and
      pixel toolbar.
    * Cursor in all input fields is now red for easier reading.
    * The FPS input widget now displays the FPS with different number of
      digits to fit the value as best it can on the limited width.

This discussion was created from the release [mrv2 v0.8.0](https://github.com/ggarra13/mrv2/releases/tag/v0.8.0)

**Comment by BigRoy** [comment]
> This discussion was created from the release [mrv2 v0.8.0](https://github.com/ggarra13/mrv2/releases/tag/).

The URL at the bottom linking to the "release" does not work.

It's missing the version's tag in the URL:

```
Current:
https://github.com/ggarra13/mrv2/releases/tag/
Should be:
https://github.com/ggarra13/mrv2/releases/tag/v0.8.0
```

**Comment by ggarra13** [comment]
Thanks for catching that.  Github creates the link automatically, but something must have wrong.
